Build failure: tigervnc

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```
config = {
    environment = {
      systemPackages = with pkgs; [
        tigervnc
      ];
    };
  };
```

### Can Hydra reproduce this build failure?

Hydra is currently rebuilding this package.

### Link to Hydra build job

https://hydra.nixos.org/build/307493610

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/62glcn77bbyn55fsd0r7iwmmdsr4lmcq-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
substituteStream() in derivation tigervnc-1.15.0: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'vncviewer/vncviewe>
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BU>
-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/95k9rsn1zsw1yvir8mj824ldhf90i4qw-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/95k9rsn1zsw1yvir8mj824ldhf90i4qw-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMAKE_BUILD_TYPE = Release
-- VERSION = 1.15.0
-- BUILD_TIMESTAMP = 1980-01-01 00:00
-- Found X11: /nix/store/xw2pibqjgmd0hfsjgvn2q0igam3fb4h5-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/wfna2fx4023769504rjzgn0mz0m6bc0c-libx11-1.8.12/lib/libX11.so;/nix/store/vqviqi56cf6wri18bscgxgx45lwb6m>
-- Looking for XOpenDisplay in /nix/store/wfna2fx4023769504rjzgn0mz0m6bc0c-libx11-1.8.12/lib/libX11.so;/nix/store/vqviqi56cf6wri18bscgxgx45lwb6m>
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found ZLIB: /nix/store/09sifcahf0j1xnw80k9l33jzcs1p2qbw-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found PkgConfig: /nix/store/05h9vfzhqf7l6w1xczixici2ldw9y788-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'pixman-1'
--   Found pixman-1, version 0.46.2
-- Found Gettext: /nix/store/b2jdp6hfygj6mxndszl0yyczprm1h8b3-gettext-0.25.1/bin/msgmerge (found version "0.25.1")
-- Performing Test ICONV_HAVE_WERROR
-- Performing Test ICONV_HAVE_WERROR - Success
-- Performing Test ICONV_SECOND_ARGUMENT_IS_CONST
-- Performing Test ICONV_SECOND_ARGUMENT_IS_CONST - Failed
-- Found Iconv: /nix/store/776irwlgfb65a782cxmyk61pck460fs9-glibc-2.40-66/lib/libc.so
-- Looking for dgettext
-- Looking for dgettext - found
-- Checking for module 'libavcodec'
--   Found libavcodec, version 61.19.101
-- Checking for module 'libavutil'
--   Found libavutil, version 59.39.100
-- Checking for module 'libswscale'
--   Found libswscale, version 8.3.100
-- Found JPEG: /nix/store/ld48pvj263k06rvljmh8la69ghh11yy0-libjpeg-turbo-3.1.1/lib/libjpeg.so (found version "62")
-- Performing Test FOUND_LIBJPEG_TURBO
-- Performing Test FOUND_LIBJPEG_TURBO - Success
-- Found FLTK: /nix/store/wi7525bbfm89l7c41rqf893j943q29qy-fltk-1.3.8/lib/libfltk_images.so;/nix/store/wi7525bbfm89l7c41rqf893j943q29qy-fltk-1.3>
-- Performing Test OK_FLTK_VERSION
-- Performing Test OK_FLTK_VERSION - Success
Package libtasn1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtasn1.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtasn1', required by 'gnutls', not found
Package libtasn1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtasn1.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtasn1', required by 'gnutls', not found
Package libtasn1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtasn1.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtasn1', required by 'gnutls', not found
Package libtasn1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtasn1.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtasn1', required by 'gnutls', not found
-- Found GnuTLS: /nix/store/a55chb1mj39mgjcbw0h655ri6dlgd0wl-gnutls-3.8.10/lib/libgnutls.so (found version "3.8.10")
-- Checking for module 'gmp'
--   Found gmp, version 6.3.0
-- Checking for module 'nettle>=3.0'
--   Found nettle, version 3.10.2
-- Checking for module 'hogweed'
--   Found hogweed, version 3.10.2
-- Looking for include file security/pam_appl.h
-- Looking for include file security/pam_appl.h - found
-- Looking for pam_start
-- Looking for pam_start - found
-- Looking for include file selinux/selinux.h
-- Looking for include file selinux/selinux.h - not found
CMake Warning at CMakeLists.txt:365 (message):
  Could not find SELinux development files


-- Checking for module 'libsystemd'
--   Found libsystemd, version 257
-- Checking for module 'pwquality'
--   No package 'pwquality' found
-- Creating static libtool control file for target os
-- Creating static libtool control file for target rdr
-- Creating static libtool control file for target network
-- Creating static libtool control file for target rfb
-- Creating static libtool control file for target unixcommon
-- Configuring done (6.8s)
-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/source
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j16 SHELL=/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/bash
[  0%] Building CXX object common/rfb/CMakeFiles/rfb.dir/AccessRights.cxx.o
[  1%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Congestion.cxx.o
[  1%] Building CXX object unix/common/CMakeFiles/unixcommon.dir/randr.cxx.o
[  2%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CConnection.cxx.o
[  3%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Blacklist.cxx.o
[  4%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CMsgReader.cxx.o
[  4%] Building CXX object unix/tx/CMakeFiles/tx.dir/TXWindow.cxx.o
[  4%] Building CXX object tests/perf/CMakeFiles/test_util.dir/util.cxx.o
[  5%] Building C object unix/vncserver/CMakeFiles/vncsession.dir/vncsession.c.o
[  5%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CMsgWriter.cxx.o
[  5%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CMsgHandler.cxx.o
[  6%] Generating bg.mo
[  7%] Generating cs.mo
[  7%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityPlain.cxx.o
[  8%] Generating vncviewer.desktop
[  7%] Generating org.tigervnc.vncviewer.metainfo.xml
[  9%] Generating da.mo
[  9%] Generating de.mo
[ 10%] Generating el.mo
[ 10%] Generating eo.mo
[ 10%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityStack.cxx.o
[ 11%] Generating es.mo
[ 11%] Generating fi.mo
[ 11%] Generating fr.mo
[ 11%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityVeNCrypt.cxx.o
[ 12%] Generating fur.mo
[ 12%] Generating he.mo
[ 12%] Built target appstream
[ 12%] Built target desktop
/build/source/unix/vncserver/vncsession.c: In function 'run_script':
/build/source/unix/vncserver/vncsession.c:549:9: warning: ignoring return value of 'chdir' declared with attribute 'warn_unused_result' [-Wunuse>
  549 |         chdir("/");
      |         ^~~~~~~~~~
[ 12%] Linking CXX static library libtest_util.a
[ 13%] Generating hu.mo
/build/source/unix/vncserver/vncsession.c: In function 'finish_daemon':
/build/source/unix/vncserver/vncsession.c:137:5: warning: ignoring return value of 'write' declared with attribute 'warn_unused_result' [-Wunuse>
  137 |     write(daemon_pipe_fd, "+", 1);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 13%] Generating id.mo
[ 14%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityVncAuth.cxx.o
[ 15%] Generating it.mo
/build/source/po/it.po:3: warning: header field 'PO-Revision-Date' missing in header
/build/source/po/it.po:3: warning: header field 'Last-Translator' missing in header
/build/source/po/it.po:3: warning: header field 'Language-Team' missing in header
[ 15%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ClientParams.cxx.o
[ 15%] Built target test_util
[ 15%] Generating ka.mo
[ 16%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ComparingUpdateTracker.cxx.o
[ 17%] Generating ko.mo
[ 17%] Generating nl.mo
[ 17%] Generating pl.mo
[ 18%] Generating pt_BR.mo
[ 18%] Generating ro.mo
[ 19%] Generating ru.mo
[ 19%] Generating sk.mo
[ 20%] Generating sr.mo
[ 20%] Generating sv.mo
[ 21%] Generating tr.mo
[ 21%] Generating uk.mo
[ 21%] Generating vi.mo
[ 22%] Generating zh_CN.mo
[ 22%] Generating zh_TW.mo
[ 22%] Built target translations
[ 22%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Configuration.cxx.o
[ 22%] Linking C executable vncsession
[ 23%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CopyRectDecoder.cxx.o
[ 23%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Cursor.cxx.o
[ 24%] Building CXX object common/rfb/CMakeFiles/rfb.dir/DecodeManager.cxx.o
[ 24%] Built target vncsession
[ 24%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Decoder.cxx.o
[ 24%] Building C object common/rfb/CMakeFiles/rfb.dir/d3des.c.o
[ 25%] Building CXX object common/rfb/CMakeFiles/rfb.dir/EncodeManager.cxx.o
[ 25%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Encoder.cxx.o
[ 26%] Building CXX object common/rfb/CMakeFiles/rfb.dir/HextileDecoder.cxx.o
[ 26%] Building CXX object common/rfb/CMakeFiles/rfb.dir/HextileEncoder.cxx.o
[ 27%] Building CXX object common/rfb/CMakeFiles/rfb.dir/JpegCompressor.cxx.o
[ 27%] Building CXX object common/rfb/CMakeFiles/rfb.dir/JpegDecompressor.cxx.o
[ 28%] Linking CXX static library libtx.a
[ 29%] Building CXX object common/rfb/CMakeFiles/rfb.dir/KeyRemapper.cxx.o
[ 29%] Building C object common/rfb/CMakeFiles/rfb.dir/KeysymStr.c.o
[ 29%] Building CXX object common/rfb/CMakeFiles/rfb.dir/LogWriter.cxx.o
[ 29%] Built target tx
[ 30%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Logger.cxx.o
[ 30%] Linking CXX static library libunixcommon.a
[ 30%] Built target unixcommon
[ 30%] Updating timestamp on libunixcommon.la
[ 31%] Creating symlink .libs/libunixcommon.a
[ 31%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Logger_file.cxx.o
[ 31%] Built target unixcommon.la
[ 32%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Logger_stdio.cxx.o
[ 32%] Building CXX object common/rfb/CMakeFiles/rfb.dir/PixelBuffer.cxx.o
[ 33%] Building CXX object common/rfb/CMakeFiles/rfb.dir/PixelFormat.cxx.o
[ 33%] Building CXX object common/rfb/CMakeFiles/rfb.dir/RREEncoder.cxx.o
[ 34%] Building CXX object common/rfb/CMakeFiles/rfb.dir/RREDecoder.cxx.o
[ 34%] Building CXX object common/rfb/CMakeFiles/rfb.dir/RawDecoder.cxx.o
[ 34%] Building CXX object common/rfb/CMakeFiles/rfb.dir/RawEncoder.cxx.o
[ 35%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Region.cxx.o
[ 35%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SConnection.cxx.o
[ 36%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SMsgHandler.cxx.o
[ 36%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SMsgReader.cxx.o
[ 37%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SMsgWriter.cxx.o
[ 37%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ServerCore.cxx.o
[ 38%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ServerParams.cxx.o
[ 38%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Security.cxx.o
[ 38%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SecurityServer.cxx.o
[ 39%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SecurityClient.cxx.o
[ 39%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityPlain.cxx.o
[ 40%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityStack.cxx.o
[ 40%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityVncAuth.cxx.o
[ 41%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityVeNCrypt.cxx.o
[ 41%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Timer.cxx.o
[ 42%] Building CXX object common/rfb/CMakeFiles/rfb.dir/TightDecoder.cxx.o
[ 42%] Building CXX object common/rfb/CMakeFiles/rfb.dir/TightEncoder.cxx.o
[ 42%] Building CXX object common/rfb/CMakeFiles/rfb.dir/TightJPEGEncoder.cxx.o
[ 43%] Building CXX object common/rfb/CMakeFiles/rfb.dir/UpdateTracker.cxx.o
[ 43%] Building CXX object common/rfb/CMakeFiles/rfb.dir/VNCSConnectionST.cxx.o
[ 44%] Building CXX object common/rfb/CMakeFiles/rfb.dir/VNCServerST.cxx.o
[ 44%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ZRLEEncoder.cxx.o
[ 45%] Building CXX object common/rfb/CMakeFiles/rfb.dir/ZRLEDecoder.cxx.o
[ 45%] Building CXX object common/rfb/CMakeFiles/rfb.dir/encodings.cxx.o
[ 45%] Building CXX object common/rfb/CMakeFiles/rfb.dir/obfuscate.cxx.o
[ 46%] Building CXX object common/rfb/CMakeFiles/rfb.dir/util.cxx.o
[ 46%] Building CXX object common/rfb/CMakeFiles/rfb.dir/H264Decoder.cxx.o
[ 47%] Building CXX object common/rfb/CMakeFiles/rfb.dir/H264DecoderContext.cxx.o
[ 47%] Building CXX object common/rfb/CMakeFiles/rfb.dir/H264LibavDecoderContext.cxx.o
[ 48%] Building CXX object common/rfb/CMakeFiles/rfb.dir/Logger_syslog.cxx.o
[ 48%] Building CXX object common/rfb/CMakeFiles/rfb.dir/UnixPasswordValidator.cxx.o
[ 49%] Building C object common/rfb/CMakeFiles/rfb.dir/pam.c.o
[ 49%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityTLS.cxx.o
[ 49%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityTLS.cxx.o
[ 50%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityDH.cxx.o
[ 50%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityMSLogonII.cxx.o
[ 51%] Building CXX object common/rfb/CMakeFiles/rfb.dir/CSecurityRSAAES.cxx.o
[ 51%] Building CXX object common/rfb/CMakeFiles/rfb.dir/SSecurityRSAAES.cxx.o
[ 52%] Linking CXX static library librfb.a
[ 52%] Built target rfb
[ 53%] Building CXX object common/network/CMakeFiles/network.dir/Socket.cxx.o
[ 54%] Building CXX object common/network/CMakeFiles/network.dir/UnixSocket.cxx.o
[ 54%] Building CXX object common/network/CMakeFiles/network.dir/TcpSocket.cxx.o
[ 54%] Linking CXX static library libnetwork.a
[ 54%] Built target network
[ 54%] Building CXX object common/rdr/CMakeFiles/rdr.dir/AESInStream.cxx.o
[ 54%] Building CXX object common/rdr/CMakeFiles/rdr.dir/BufferedInStream.cxx.o
[ 54%] Building CXX object common/rdr/CMakeFiles/rdr.dir/BufferedOutStream.cxx.o
[ 55%] Building CXX object common/rdr/CMakeFiles/rdr.dir/FdOutStream.cxx.o
[ 55%] Building CXX object common/rdr/CMakeFiles/rdr.dir/FileInStream.cxx.o
[ 56%] Building CXX object common/rdr/CMakeFiles/rdr.dir/Exception.cxx.o
[ 57%] Building CXX object common/rdr/CMakeFiles/rdr.dir/AESOutStream.cxx.o
[ 57%] Building CXX object common/rdr/CMakeFiles/rdr.dir/FdInStream.cxx.o
[ 58%] Building CXX object common/rdr/CMakeFiles/rdr.dir/HexInStream.cxx.o
[ 59%] Building CXX object common/rdr/CMakeFiles/rdr.dir/RandomStream.cxx.o
[ 59%] Building CXX object common/rdr/CMakeFiles/rdr.dir/HexOutStream.cxx.o
[ 60%] Building CXX object common/rdr/CMakeFiles/rdr.dir/TLSOutStream.cxx.o
[ 60%] Building CXX object common/rdr/CMakeFiles/rdr.dir/ZlibInStream.cxx.o
[ 61%] Building CXX object common/rdr/CMakeFiles/rdr.dir/ZlibOutStream.cxx.o
[ 60%] Building CXX object common/rdr/CMakeFiles/rdr.dir/TLSInStream.cxx.o
[ 61%] Building CXX object common/rdr/CMakeFiles/rdr.dir/TLSException.cxx.o
[ 61%] Linking CXX static library librdr.a
[ 61%] Built target rdr
[ 62%] Building CXX object common/os/CMakeFiles/os.dir/Mutex.cxx.o
[ 62%] Building CXX object common/os/CMakeFiles/os.dir/Thread.cxx.o
[ 62%] Building CXX object common/os/CMakeFiles/os.dir/os.cxx.o
[ 63%] Linking CXX static library libos.a
[ 63%] Built target os
[ 64%] Updating timestamp on libos.la
[ 64%] Creating symlink .libs/libos.a
[ 65%] Updating timestamp on libnetwork.la
[ 66%] Updating timestamp on librdr.la
[ 67%] Creating symlink .libs/librfb.a
[ 67%] Updating timestamp on librfb.la
[ 67%] Creating symlink .libs/librdr.a
[ 67%] Creating symlink .libs/libnetwork.a
[ 67%] Building C object unix/x0vncserver/CMakeFiles/x0vncserver.dir/buildtime.c.o
[ 68%] Building CXX object unix/vncpasswd/CMakeFiles/vncpasswd.dir/vncpasswd.cxx.o
[ 69%] Building CXX object tests/perf/CMakeFiles/encperf.dir/encperf.cxx.o
[ 71%] Building C object unix/vncconfig/CMakeFiles/vncconfig.dir/buildtime.c.o
[ 71%] Building CXX object tests/perf/CMakeFiles/convperf.dir/convperf.cxx.o
[ 72%] Building CXX object tests/perf/CMakeFiles/decperf.dir/decperf.cxx.o
[ 71%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/fltk/Fl_Monitor_Arrangement.cxx.o
[ 72%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/fltk/Fl_Navigation.cxx.o
[ 72%] Built target rdr.la
[ 72%] Building C object unix/vncconfig/CMakeFiles/vncconfig.dir/vncExt.c.o
[ 73%] Building CXX object unix/vncconfig/CMakeFiles/vncconfig.dir/vncconfig.cxx.o
[ 73%] Built target network.la
[ 73%] Built target rfb.la
[ 73%] Building CXX object tests/perf/CMakeFiles/fbperf.dir/fbperf.cxx.o
[ 74%] Building CXX object tests/perf/CMakeFiles/fbperf.dir/__/__/vncviewer/PlatformPixelBuffer.cxx.o
[ 74%] Built target os.la
[ 74%] Building CXX object unix/vncconfig/CMakeFiles/vncconfig.dir/QueryConnectDialog.cxx.o
[ 75%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/Geometry.cxx.o
[ 76%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/fltk/theme.cxx.o
[ 76%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/menukey.cxx.o
[ 76%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/BaseTouchHandler.cxx.o
[ 76%] Building CXX object tests/unit/CMakeFiles/configargs.dir/configargs.cxx.o
[ 76%] Linking CXX executable convperf
[ 77%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/CConn.cxx.o
[ 77%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/DesktopWindow.cxx.o
[ 78%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/EmulateMB.cxx.o
[ 78%] Linking CXX executable vncpasswd
[ 78%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/Image.cxx.o
[ 78%] Built target convperf
[ 78%] Building CXX object tests/perf/CMakeFiles/fbperf.dir/__/__/vncviewer/Surface.cxx.o
[ 79%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/PollingManager.cxx.o
[ 80%] Building CXX object tests/perf/CMakeFiles/fbperf.dir/__/__/vncviewer/Surface_X11.cxx.o
[ 80%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/UserDialog.cxx.o
[ 81%] Linking CXX executable configargs
[ 81%] Building CXX object tests/unit/CMakeFiles/conv.dir/conv.cxx.o
[ 82%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/ServerDialog.cxx.o
[ 82%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/PollingScheduler.cxx.o
[ 82%] Building CXX object tests/unit/CMakeFiles/convertlf.dir/convertlf.cxx.o
[ 82%] Built target vncpasswd
[ 83%] Building CXX object tests/unit/CMakeFiles/gesturehandler.dir/gesturehandler.cxx.o
[ 83%] Building CXX object tests/unit/CMakeFiles/gesturehandler.dir/__/__/vncviewer/GestureHandler.cxx.o
[ 83%] Linking CXX executable decperf
[ 84%] Linking CXX executable conv
[ 84%] Built target configargs
[ 84%] Linking CXX executable encperf
[ 84%] Building CXX object tests/unit/CMakeFiles/hostport.dir/hostport.cxx.o
[ 85%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/TimeMillis.cxx.o
[ 85%] Linking CXX executable fbperf
[ 85%] Linking CXX executable vncconfig
[ 85%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/Surface.cxx.o
[ 85%] Built target decperf
[ 85%] Built target conv
[ 85%] Building CXX object tests/unit/CMakeFiles/pixelformat.dir/pixelformat.cxx.o
[ 86%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/OptionsDialog.cxx.o
[ 86%] Building CXX object tests/unit/CMakeFiles/unicode.dir/unicode.cxx.o
[ 86%] Building C object unix/x0vncserver/CMakeFiles/x0vncserver.dir/qnum_to_xorgevdev.c.o
[ 86%] Built target encperf
[ 87%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/x0vncserver.cxx.o
[ 87%] Building C object unix/x0vncserver/CMakeFiles/x0vncserver.dir/qnum_to_xorgkbd.c.o
[ 87%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/XPixelBuffer.cxx.o
[ 87%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/PlatformPixelBuffer.cxx.o
[ 87%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/Viewport.cxx.o
[ 87%] Built target vncconfig
[ 87%] Building CXX object tests/unit/CMakeFiles/emulatemb.dir/emulatemb.cxx.o
[ 88%] Linking CXX executable convertlf
[ 88%] Built target fbperf
[ 88%] Linking CXX executable hostport
[ 89%] Building CXX object tests/unit/CMakeFiles/emulatemb.dir/__/__/vncviewer/EmulateMB.cxx.o
[ 90%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/parameters.cxx.o
[ 90%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/touch.cxx.o
[ 91%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/MonitorIndicesParameter.cxx.o
[ 92%] Linking CXX executable pixelformat
[ 93%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/XDesktop.cxx.o
[ 93%] Built target convertlf
[ 93%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/vncviewer.cxx.o
[ 93%] Building C object unix/x0vncserver/CMakeFiles/x0vncserver.dir/RandrGlue.c.o
[ 93%] Built target pixelformat
[ 94%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/XSelection.cxx.o
[ 94%] Built target hostport
[ 94%] Linking CXX executable unicode
[ 95%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/GestureHandler.cxx.o
[ 95%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/XInputTouchHandler.cxx.o
[ 96%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/KeyboardX11.cxx.o
[ 96%] Linking CXX executable emulatemb
[ 96%] Building CXX object unix/x0vncserver/CMakeFiles/x0vncserver.dir/__/vncconfig/QueryConnectDialog.cxx.o
[ 96%] Built target unicode
[ 96%] Building C object vncviewer/CMakeFiles/vncviewer.dir/xkb_to_qnum.c.o
[ 96%] Building CXX object vncviewer/CMakeFiles/vncviewer.dir/Surface_X11.cxx.o
[ 96%] Built target emulatemb
/build/source/vncviewer/vncviewer.cxx: In function 'void createTunnel(const char*, const char*, int, int)':
/build/source/vncviewer/vncviewer.cxx:611:9: warning: ignoring return value of 'int system(const char*)' declared with attribute 'warn_unused_re>
  611 |   system(cmd2);
      |   ~~~~~~^~~~~~
[ 97%] Linking CXX executable gesturehandler
[ 98%] Linking CXX executable x0vncserver
[100%] Linking CXX executable vncviewer
[100%] Built target gesturehandler
[100%] Built target x0vncserver
[100%] Built target vncviewer
/build/source/unix/xserver /build/source
patching file configure.ac
Hunk #2 succeeded at 1729 (offset 8 lines).
Hunk #3 succeeded at 2060 (offset 8 lines).
Hunk #4 succeeded at 2367 (offset 8 lines).
patching file hw/Makefile.am
patching file include/dix-config.h.in
Hunk #1 succeeded at 57 (offset -6 lines).
/nix/store/jrw7q6v8q74hhv43zgpq7i4jmxj9nwlj-stdenv-linux/setup: line 267: autoreconf: command not found
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.5, NixOS, 25.11 (Xantusia), 25.11.20250908.b599843`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - channels(root): `""`
 - nixpkgs: `/nix/store/5izw1shpjxb9qhlf67bx427cih5czj8w-source`

### Notify maintainers

Unmaintained :-(

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ti/tigervnc/package.nix
+++ b/pkgs/by-name/ti/tigervnc/package.nix
@@ -22,6 +22,8 @@
   makeWrapper,
   nixosTests,
   ffmpeg,
+  autoconf,
+  automake,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
@@ -160,6 +162,8 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     cmake
     gettext
+    autoconf
+    automake
   ]
   ++ lib.optionals stdenv.hostPlatform.isLinux (
     with xorg;

